docs/detections/prebuilt-rules/rule-details/linux-restricted-shell-breakout-via-the-vi-command.asciidoc

```diff
@@ -0,0 +1,65 @@
+[[linux-restricted-shell-breakout-via-the-vi-command]]
+=== Linux Restricted Shell Breakout via the vi command
+
+Identifies Linux binary find abuse to break out from restricted environments by spawning an interactive system shell. The vi/vim is the standard text editor in Linux distribution and the activity of spawning a shell is not a standard use of this binary by a user or system administrator and could potentially indicate malicious actor attempting to improve the capabilities or stability of their access."
```

Comment:
```suggestion
Identifies Linux binary find abuse to break out from restricted environments by spawning an interactive system shell. The vi/vim editor is the standard text editor in Linux distributions, and the activity of spawning a shell is not a standard use of this binary by a user or system administrator. This could potentially indicate a malicious actor attempting to improve the capabilities or stability of their access.
```